pkg/operator2/operator.go

```diff
@@ -24,20 +25,20 @@ import (
 )
 
 const (
-	targetName = "openshift-osin"
-
 	metadataKey = "metadata"
 	configKey   = "config.yaml"
 	sessionKey  = "session"
 	sessionPath = "/var/session"
-
-	configName      = "cluster"
-	configNamespace = "openshift-managed-config"
-
-	authOperatorConfigResourceName = "cluster"
 )
 
 type authOperator struct {
+	targetName             string
```

Comment:
Seriously?  These are global constants than can never change.  Doing so would break every operator.  They are not config options.

Reply:
I disagree. There is nothing inherent in the operator that depends on what the names specifically are. It is an operational concern that gains it significance from our particular cluster setup. There are also a lot of problems with depending on global state across the package. It is much safer and easier to reason about explicitly defined scopes.

Reply:
No.  Constants remain constants.  I can find their usages and reason about them fine.  They will not be converted into mutable state inside a struct that I have to care about.  I do not need nor want any flexibility on these names.  Scoping the state inside the struct makes it harder to reason about, not easier.